lib/mcl.ts

```diff
@@ -107,6 +106,10 @@ export class Fr {
         this.mclFr.setByCSPRNG();
     }
 
+    setFromArray(arr: Uint8Array) {
```

Comment:
Good point. I am going to rename `setFromArray` to `setLittleEndianMod`. I was thinking of handling the endianess here, but it might not be the best place, indeed.